python/tvm/relay/frontend/pytorch.py

```diff
@@ -5871,7 +5871,19 @@ def export_c_graph(location, graph):
     fname = os.path.join(location, f"tvm_exported_c_graph_{time_stamp}.txt")
     with open(f"{fname}", "w") as f:
         f.write(str(graph))
-        
+
+def _binray_search(lst, func):
```

Comment:
Correct spelling to `_binary_search(lst, func):`

Reply:
Thanks for finding it. Fixed